packages/app-mobile/components/screens/Note/Note.tsx

```
@@ -180,12 +180,12 @@ class NoteScreenComponent extends BaseScreenComponent<ComponentProps, State> imp
 	private editorRef: any;
 	private titleTextFieldRef: RefObject<TextInput>;
 	private navHandler: OnNavigateCallback;
-	private backHandler: ()=> Promise<boolean>;
+	private backHandler: () => Promise<boolean>;
 	private undoRedoService_: UndoRedoService;
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
 	private noteTagDialog_closeRequested: any;
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	private refreshResource: (resource: any, noteBody?: string)=> Promise<void>;
+	private refreshResource: (resource: any, noteBody?: string) => Promise<void>;
```

ESLint

[error] 'resource' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)